graphql_compiler/compiler/metadata.py

```diff
@@ -132,6 +154,27 @@ def get_location_info(self, location):
                                  u'{}'.format(location))
         return location_info
 
+    def record_filter_info(self, location, filter_info):
+        """Record filter information about the location."""
+        if isinstance(location, FoldScopeLocation):
+            # NOTE(gurer): ignore filters inside the fold for now
```

Comment:
@gurer-kensho if you'd like a suggestion for a following PR, you could simply add an `at_vertex()` method on `FoldScopeLocation` and then delete this check. That method doesn't exist because there wasn't a use for it, and not because it doesn't make sense to have it -- `FoldScopeLocation` objects can point to fields just as much as vanilla `Location` objects can.

Reply:
We decided on doing this in a subsequent PR, right?